server/src/test/java/com/objectcomputing/checkins/services/pulseresponse/PulseResponseControllerTest.java

```diff
@@ -20,11 +20,7 @@
 import org.junit.jupiter.api.Test;
 
 import java.time.LocalDate;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
-import java.util.Set;
-import java.util.UUID;
+import java.util.*;
```

Comment:
Do we prefer wildcard imports over specific named ones?  I'm new here, so need to learn 😄

Reply:
@mkimberlin thoughts?

Reply:
Reverting to what was there.